doc/examples/streamline_tools.py

```diff
@@ -48,7 +48,7 @@
 label_fname = get_fnames(name="stanford_labels")
 t1_fname = get_fnames(name="stanford_t1")
 
-data, affine, hardi_img = load_nifti(hardi_fname, return_img=True)
+data, _, hardi_img = load_nifti(hardi_fname, return_img=True)
```

Comment:
LGTM, but isnot it strange that we do not use the affine in this tutorial? why we use `np.eye` instead ?

Reply:
It is Serge. The only thing I investigated is whether the dataset affine is the identity matrix, which is not. I agree that the issue deserves further investigation.

Reply:
If we end up finding that this is a bug and that this very same affine needs to be used, we'll need to move and adapt the text about the affine added in #3375.